packages/itwinui-react/src/core/InputWithDecorations/InputWithDecorations.tsx

```diff
@@ -79,14 +79,14 @@ const InputWithDecorationsButton = React.forwardRef((props, ref) => {
  * Input component with various additional decorations.
  * You can add icons, buttons and other various subcomponents to it.
  *
- * If you are not using default InputWithDecorations.Icon and InputWithDecorations.Button, use borderless versions of other components.
+ * If you are not using default `Icon` and {@link InputWithDecorations.Button}, use borderless versions of other components.
```

Comment:
nit: These `@link` things do not show up in docs, so it's better to instead use markdown syntax (or plain text). I think it's also not usable by developers who are not working within the iTwinUI codebase.

<img width="649" alt="image" src="https://github.com/iTwin/iTwinUI/assets/9084735/a915839d-e4e2-4fa6-9f65-0489e62d117a">

Reply:
Sounds good, removed (https://github.com/iTwin/iTwinUI/pull/1876/commits/70d411e741a6b2c8153c2ef09f5a262e0b582a92).